app/models/partner.rb

```diff
@@ -13,7 +13,26 @@ def ensure_role
     self.role = 'partner'
   end
 
+  def balance
+    wallet.contribution_balance.amount_bat + invoice_amount
+  end
+
+  def balance_in_currency
+    wallet.contribution_balance.add_bat(invoice_amount)
+
+    wallet.contribution_balance.amount_default_currency
+  end
+
   def name
     self[:name] || self[:email]
   end
+
+  private
+
+  def invoice_amount
+    invoices = Invoice.where(partner_id: id, paid: false)
+    amounts = invoices.map { |i| i.finalized_amount || i.amount }
+
+    amounts.map { |x| x.gsub(/,/, '').to_i }.reduce(:+)
```

Comment:
Quick question, do we prefer .gsub or trim? I've been having good results with 
```ruby
str.tr(",", "")
```

But I'm not sure which is better

Reply:
https://github.com/brave-intl/publishers/pull/1667/commits/092d85279ad37ad6a8ce82db79c7972594e84ead